.github/workflows/fossa.yml

```diff
@@ -4,28 +4,20 @@ on:
   push:
     branches: 
       - main
-  pull_request:
-    branches:
-      - main
 
 permissions:
   contents: read
 
 jobs:
-  build:
+  fossa-scan:
+    if: ${{ secrets.FOSSA_API_KEY }}
```

Comment:
Is this key available already or you added a new key?

Reply:
I added it.

Reply:
ok thanks!